static/app/views/replays/detail/network/index.tsx

```diff
@@ -192,7 +192,7 @@ function NetworkList() {
 
   const currentIndex = indexAtCurrentTime();
   const showJumpDownButton = currentIndex > visibleRange[1];
-  const showJumpUpButton = currentIndex < visibleRange[0];
+  const showJumpUpButton = currentIndex < visibleRange[0] && networkFrames?.length;
```

Comment:
should we add the condition to the down button too? just in case

Reply:
it could also be combined with `sortConfig.by === 'startTimestamp'` and given a new var name